# estimated centripetal acceleration and tangential velocity
float32[3] aE		# accel in earth frame
float32 centripMag	# magnitude of A_i
float32 thetaT		# thrust vector rotation rate
float32 omegaE		# earth frame rotation rate
float32 tV			# estimated tangential velocity
float32[3] estG		# estimated g vector in earth frame
float32[3] centripA	# estimated centrip. acc vector
float32[3] mag_n	# normalized mag vector in body frame		
float32[3] mag_err	# body frame mag error
float32[3] magRef	# magnetic vector in earth frame
